modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator_{context}"]
+= Installing the AWS Load Balancer Operator
+
+You can install the AWS Load Balancer Operator using the Red Hat {product-title} Operator Hub.
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+.Procedure
+
+. Click *Operators* → *OperatorHub* in the OpenShift Web Console.
+. Click *AWS Load Balancer Operator*.
+You can use the Filter by keyword text box or the filter list to search for AWS Load Balancer Operator from the list of operators.
+. Select `` namespace.
+. On the AWS Load Balancer Operator page, click *Install*.
+. On the Install Operator page, ensure that you selected the following options:
+.. Update the channel as *stable-{product-version}*.
```

Comment:
The product version won't be the right, that's I'm pretty sure. I'm wondering also about the naming of the channel for the TechPreview. What's the plan for it, @arjunrn ?

Reply:
@xenolinux please take note. The product version of this operator is not the same as the OCP version. So this line should be:
```suggestion
.. Update the channel as *stable-v0.1*.
```
@alebedev87 I have a [PR](openshift/aws-load-balancer-operator#60) to change the channel name.

Reply:
Ack'ed